azure-pipelines.yml

```diff
@@ -0,0 +1,49 @@
+# This pipeline is NEVER triggered automatically.
+# This pipeline is used for initialization of a new (or cleared) kubernetes cluster.
+# Running this pipeline unnecessarily may cause the ingress IP address to reset.
+# The ingress IP address is how you access the application, so...
+# BE CAREFUL!!!
+trigger: none
```

Comment:
1. writing a comment like that you, were just setting yourself up... this pipeline as-is runs automatically on every PR. (the default pr trigger is equivalent to `pr: '*'`)
2. ingress IPs changing isn't a problem if you use the azure dns annotation to get a permanent domain for the ingress.
3. kubernetes is declarative, so there's no reason we wouldn't want to apply the prerequisite manifests automatically to both clusters on this repo's main branch on any change to them. even the ingress IP wouldn't change just because you re-applied the same manifest.
4. this doesn't seem to fully set up the cluster, since i don't see any ingress controller being configured. unless istio just does that automatically. at the end of the day, we want things like that documented, maybe not in this PR when things are in flux.

Reply:
We may re-examine this, but for now, I added `pr: none` to block automated builds.

Also, the Istio has the ingress controller and is being setup cluster-wide.  We need to add that to the documentation, but I am pretty sure that Sam has it in his pipeline for the whole cluster.